Documents/Development/Contributing-Guidelines.md

```diff
@@ -1,3 +1,80 @@
 # Contributing Guidelines for Krypton Development
 
-BEFORE CONTRIBUTING please read
\ No newline at end of file
+BEFORE CONTRIBUTING please read _all_ of the rules below are guidelines, which means that they should be followed when possible. Please do not take them literally.
```

Comment:
If they are guidelines, then they are not rules, Confusing text.

Reply:
Cleared up paragraph text